app/editor/src/features/content/form/components/tool-bar/form/PublishedSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { IContentForm } from 'features/content/form/interfaces';
+import React from 'react';
+import { FaExternalLinkAlt } from 'react-icons/fa';
+import { toast } from 'react-toastify';
+import { useSettings } from 'store/hooks/admin';
+import { Col, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IPublishedSectionProps {
+  /** Form values */
+  values: IContentForm;
+}
+
+export const PublishedSection: React.FC<IPublishedSectionProps> = ({ values }) => {
+  const [subscriberUrl, setSubscriberUrl] = React.useState<string>();
+  const [{ settings }, api] = useSettings();
+
+  React.useEffect(() => {
+    api.findAllSettings().then((data) => {
```

Comment:
You shouldn't need to make this separate request.  I believe the lookups already include these values.  Please confirm.